bitmap-ittybitty/lib/app.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const AllThatData = module.exports = function (buffer){
```

Comment:
maybe a cleaner way to write this would be: 
```javascript
module.exports = function AllThatData(){
// blah blah blah
}
```
Since functions are hoisted, this will work the same way, but it has the advantage of not hiding your module.exports statement in the middle of a line, and not having an anonymous function being stored in a variable when that's the reason why functions can be given names in the first place.